src/cloud/architecture/starter-architecture.md

```diff
@@ -0,0 +1,135 @@
+
+Because the production environment is read-only, you must make changes in the Integration environment and deploy across the architecture from the Integration environment to Staging, and finally to the Production environment. See [Deploy your store][] and [Site launch][].
+
+We highly recommend fully testing in your Staging environment and branch before pushing to the `master` branch which deploys to the Production environment.
+
+## Staging environment {#cloud-arch-stage}
+
+We recommend creating a branch called `staging` from `master`. The `staging` branch deploys code to the Staging environment to provide a pre-production environment to test code, modules and extensions, payment gateways, shipping, product data, and much more. This environment provides the configuration for all services to match the Production environment including Fastly, New Relic APM, and search.
+
+Additional sections in this guide provide instructions for final code deployments and testing production level interactions in a secure Staging environment. For best performance and feature testing, replicate your Production database into the Staging environment.
+
+{:.bs-callout-warning}
+We highly recommend testing every merchant and customer interaction in the Staging environment prior to deploying to the Production environment. See [Deploy your store][] and [Test deployment][].
+
+## Integration environment {#cloud-arch-int}
+
+Developers use the Integration environment to develop, deploy, and test:
+
+-  Magento application code
+-  Custom code
+-  Extensions
+-  Services
+
+You can have **one** active Integration environment. You create the Integration environment by creating a branch from the Staging branch. When you create an Integration environment, the environment name matches the branch name. An integration environment includes a web server and a database. It does not include all services, for example Fastly CDN and New Relic are not available.
+
+You can have an unlimited number of inactive branches for code storage. To access, view, and test an inactive branch, you must activate it.
+
+{% include cloud/note-enhanced-integration-envs-kb.md%}
+
+## Production and Staging technology stack {#technology}
+
+The Production and Staging environments include the following technologies. You can modify and configure these technologies through the [`.magento.app.yaml`][] file.
+
+-  Fastly for HTTP caching and CDN
+-  Nginx web server speaking to PHP-FPM, one instance with multiple workers
+-  Redis server
+-  Elasticsearch for searching for {{site.data.var.ece}} 2.2 and later
+
+### Services {#cloud-arch-services}
+
+{{site.data.var.ece}} currently supports the following services: PHP, MySQL (MariaDB), Elasticsearch (Magento 2.2.x and later), Redis, and RabbitMQ.
+
+Each service runs in its own secure container. Containers are managed together in the project. Some services are built-in, such as the following:
```

Comment:
I've never been a fan of "It's own". I always think of "getting out of one's own way". 

```suggestion
Each service runs in a separate, secure container. Containers are managed together in the project. Some services are standard, such as the following:
```